src/analyses/reaching_definitions.cpp

```diff
@@ -210,9 +210,12 @@ void rd_range_domaint::transform_function_call(
       if(identifier.empty())
         continue;
 
-      range_spect size=
-        to_range_spect(pointer_offset_bits(param.type(), ns));
-      gen(from, identifier, 0, size);
+      auto param_bits = pointer_offset_bits(param.type(), ns);
+      if(param_bits.has_value())
+      {
+        range_spect size = to_range_spect(*param_bits);
+        gen(from, identifier, 0, size);
+      }
```

Comment:
No, this should invoke `gen` even when the size cannot be determined (using `-1` as a value of `size` is well-defined).

Reply:
fixed.